letsencrypt-auto-source/tests/auto_test.py

```diff
@@ -1,6 +1,10 @@
 """Tests for letsencrypt-auto"""
 
-from BaseHTTPServer import HTTPServer, BaseHTTPRequestHandler
+# PY2,PY3
```

Comment:
I am surprised that this modification is needed, it does not seem related to the PR.